src/lib.rs

```diff
@@ -30,19 +28,21 @@ pub use vbe_info::{
     VBECapabilities, VBEControlInfo, VBEDirectColorAttributes, VBEField, VBEInfoTag,
     VBEMemoryModel, VBEModeAttributes, VBEModeInfo, VBEWindowAttributes,
 };
+use tags::mbi::{TagType, Tag, TagIter};
 
 #[macro_use]
 extern crate bitflags;
+extern crate alloc;
```

Comment:
I don't think we can do this - people need to parse the multiboot structures before they have a heap set up, generally. This actually caused me to check and we actually need to move this crate over to the 2018 edition... oops.

Reply:
Ah yeah, this was my IDE during development. It is not necessary anymore and can be removed.